README.md

```diff
@@ -141,3 +141,7 @@ https://docs.docker.com/storage/volumes/
 Please note that in this code base the .env file have been added to a .dockerignore file for security reasons. 
 See this link for more info regarding the use of the dockerignore file: 
 https://docs.docker.com/engine/reference/builder/#dockerignore-file
+
+## Launching the app with Heroku
+
+To see the app working in Heroku, please go to this link: https://wisti.herokuapp.com/
```

Comment:
Thanks for adding this link - note that this probably suggests your HEROKU_APP_NAME doesn't need to be a secret in the pipeline because it's given away by this!